WordPressKit/WordPressComServiceRemote+SiteCreation.swift

```diff
@@ -6,16 +6,39 @@ import Foundation
 /// This value type is intended to express a site creation request.
 ///
 public struct SiteCreationRequest: Encodable {
-    let segmentIdentifier: Int64
-    let verticalIdentifier: String?
-    let title: String
-    let tagline: String?
-    let siteURLString: String
-    let isPublic: Bool
-    let languageIdentifier: String
-    let shouldValidate: Bool
-    let clientIdentifier: String
-    let clientSecret: String
+    public let segmentIdentifier: Int64
+    public let verticalIdentifier: String?
+    public let title: String
+    public let tagline: String?
+    public let siteURLString: String
+    public let isPublic: Bool
+    public let languageIdentifier: String
+    public let shouldValidate: Bool
+    public let clientIdentifier: String
+    public let clientSecret: String
+
+    public init(segmentIdentifier: Int64,
```

Comment:
Do we need a manual `init` here? Sorry if I missed something in here, its just looks like it's the same as the autogenerated one?

Reply:
I thought so too, but alas, when attempting to instantiate `SiteCreationRequest` in WPiOS, I encountered the following compiler error:
> 'SiteCreationRequest' initializer is inaccessible due to 'internal' protection level

Reply:
Ohhhhhh right. I would've seen that from my unit test! In that case, :shipit:!